apis/v1alpha1/cluster_types.go

```diff
@@ -109,6 +109,10 @@ type CrdbClusterSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Map of additional custom labels"
 	// +optional
 	AdditionalLabels map[string]string `json:"additionalLabels,omitempty"`
+	// (Optional) Additional custom resource annotations that are added to all resources
```

Comment:
Think we should add something here to remind users that changing this will restart the cluster?

Reply:
Done